web-console/src/views/load-data-view/info-messages.tsx

```diff
@@ -216,3 +217,48 @@ export const SpecMessage = React.memo(function SpecMessage() {
     </FormGroup>
   );
 });
+
+export interface AppendToExistingIssueProps {
+  spec: Partial<IngestionSpec>;
+  onChangeSpec(newSpec: Partial<IngestionSpec>): void;
+}
+
+export const AppendToExistingIssue = React.memo(function AppendToExistingIssue(
```

Comment:
Nit: Do we want to call it `AppendToExistingIssue`?
To me, `issue` implies that there is a known bug and we are yet to fix it.

Reply:
Within the context of the console that is exactly what it is... the user has selected UI options that put the spec in the data loader into a state that can not be submitted. And this callout prompts the user to fix it. Note that I am not making a judgment on whether this behavior is an "issue in Druid" just that the data loader has an internal issue.

The term "issue" is used in the same context all over the console codebase, for example https://github.com/apache/druid/blob/master/web-console/src/druid-models/ingestion-spec/ingestion-spec.tsx#L240 - this function will flag an "issue" if `spec.dataSchema.dataSource` is not set.

Reply:
Hmm, I see. Thanks for the clarification 👍🏻